repose-aggregator/external/service-clients/auth/pom.xml

```diff
@@ -46,6 +46,12 @@
         </dependency>
 
         <dependency>
+            <groupId>org.openrepose</groupId>
```

Comment:
It brought in FilterDirector.SC_TOO_MANY_REQUESTS.
I can remove it and leave it with the 429 as a magic number or define it locally.